ecc/Key.py

```diff
@@ -294,27 +297,25 @@ def auth_decrypt(self, data, source):
         if source.verify(text, sgn):
             return text
         else:
-            raise SecurityViolationException, "Invalid Signature"
+            raise SecurityViolationException, 'Invalid Signature'
 
 
-if __name__ == "__main__":
+if __name__ == '__main__':
 
     import time
 
     def test_overhead():
-        print "sender", "receiver", "+bytes", "+enctime", "+dectime"
+        print('sender', 'receiver', '+bytes', '+enctime', '+dectime')
         for s in [192, 224, 256, 384, 521]:
             sender = Key.generate(s)
             for r in [192, 224, 256, 384, 521]:
                 receiver = Key.generate(r)
                 t = time.time()
-                e = sender.auth_encrypt("", receiver)
+                e = sender.auth_encrypt('', receiver)
                 t1 = time.time() - t
                 t = time.time()
                 receiver.auth_decrypt(e, sender)
                 t2 = time.time() - t
-                print s, r, len(e), t1, t2
+                print(s, r, len(e), t1, t2)
```

Comment:
This does not work out in Python 2. Use 

    print("{} {} {} {} {}".format(s, r, len(e), t1, t2))

However, it may not be of interest, if all elements are of basic types.